x-pack/plugins/ingest_manager/public/applications/ingest_manager/components/enrollment_instructions/manual/index.tsx

```diff
@@ -29,70 +29,63 @@ export const ManualInstructions: React.FunctionComponent<Props> = ({
   const enrollArgs = `${kibanaUrl} ${apiKey.api_key}${
     kibanaCASha256 ? ` --ca_sha256=${kibanaCASha256}` : ''
   }`;
-  const macOsLinuxTarCommand = `./elastic-agent enroll ${enrollArgs}
-./elastic-agent run`;
 
-  const linuxDebRpmCommand = `elastic-agent enroll ${enrollArgs}
-systemctl enable elastic-agent
-systemctl start elastic-agent`;
+  const linuxMaxCommand = `./elastic-agent install -f ${enrollArgs}`;
 
-  const windowsCommand = `.\\elastic-agent enroll ${enrollArgs}
-.\\install-service-elastic-agent.ps1`;
+  const windowsCommand = `.\\elastic-agent.exe install -f ${enrollArgs}`;
 
   return (
     <>
       <EuiText>
         <FormattedMessage
           id="xpack.ingestManager.enrollmentInstructions.descriptionText"
-          defaultMessage="From the agent directory, run these commands to enroll and start an Elastic Agent. You can reuse these commands to set up agents on more than one host. Requires administrator privileges."
+          defaultMessage="From the agent’s directory, run the appropriate command to install, enroll and start an Elastic Agent. You can reuse these commands to setup agents on more than one machine. Be sure to run the install steps as a user with Administrator privilege on the system."
```

Comment:
```suggestion
          defaultMessage="From the agent directory, run the appropriate command to install, enroll, and start an Elastic Agent. You can reuse these commands to set up agents on more than one host. Requires administrator privileges."
```

I'm not sure that we need to change the wording here that much from the original so this is my suggestion

Reply:
Looks good to me 👍